hikari/errors.py

```diff
@@ -478,6 +478,16 @@ class RateLimitTooLongError(HTTPError):
     period: float = attr.ib()
     """How long the rate limit window lasts for from start to end."""
 
+    message: str = attr.ib(init=False)
+    """The error message."""
+
+    @message.default
+    def _(self) -> str:
+        return (
```

Comment:
if we're defaulting this to a specific case then should we also specify the message for different case where this is raised such as out of bucket ratelimits to avoid confusion?